backend/routers/psOne.js

```diff
@@ -29,9 +29,17 @@ router.get('/api/1', async (req, res) => {
     queries.location = { $regex: new RegExp(req.query.location, 'i') };
   }
 
+  if (req.query.industryDomain) {
+    queries.location = { $regex: new RegExp(req.query.industryDomain, 'i') };
+  }
+
+  if (req.query.branch) {
+    queries.location = { $regex: new RegExp(req.query.branch, 'i') };
```

Comment:
queries.location?
You cannot use regex with regex with an enum

Reply:
Okay I'll read up and fix it.